hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/utils/db/TableConfig.java

```diff
@@ -30,6 +30,11 @@
  * Class that maintains Table Configuration.
  */
 public class TableConfig {
+  static TableConfig newTableConfig(byte[] bytes) {
+    return new TableConfig(StringUtils.bytes2String(bytes),
+        DBStoreBuilder.HDDS_DEFAULT_DB_PROFILE.getColumnFamilyOptions());
+  }
```

Comment:
ColumnFamilyOptions needs to be closed manually. How about making TableConfig Closeable and close the ColumnFamilyOptions object in the close() method?

Reply:
That is a good idea, will follow up in a separate PR.